### PR TITLE
Feature/mechanism of occurrence windows account

### PR DESCRIPTION
**Pull request by rshulyar**

### CODE REVIEW

**Comment by rshulyar** [comment]
https://rshulyar.atlassian.net/browse/GH-10

File: .firebase/hosting.YnVpbGQ.cache [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-static/js/main.065bae7e.chunk.js.map,1631007813016,972d8b6e2f280c18180c5878f8eab18d112cef1a60715a0196c5fbcace20bafe
-manifest.json,1631007798261,341d52628782f8ac9290bbfc43298afccb47b7cbfcee146ae30cf0f46bc30900
-static/js/main.065bae7e.chunk.js,1631007813016,ddb4c4e5c787db65379ae680a3ec5304811ce33434acbde96a0c0278e8817236
-index.html,1631007813008,60b5322d067fbb126e65d3ea1ae287bb193e76b8392e176b919438b551d43a08
-static/js/runtime-main.c8ad0dd5.js.map,1631007813017,f226916b6ec2f98cd0a4f355fd7e80fc17ce6c7ed4edf7715afeaec561825327
-static/js/runtime-main.c8ad0dd5.js,1631007813015,d58bf78c19dcf5650a61647e360806afc508292446737f35b61562923de06d8e
-static/media/menuWindowElementImages.9487e326.png,1631007813015,3f1e1da8e4d875a38c39179a751508a2a58c00a8f98079b68f1ef8a300f8f5e5
-static/js/2.af4d17ac.chunk.js,1631007813017,c9654f84af735187bd8841228de1b89792341530cbdc0a656779ff46577f739c
-static/js/2.af4d17ac.chunk.js.map,1631007813017,b35f75d066d0619acd73ff3252f6a23713e7916b00e10ca50bda01ff17f33679
+asset-manifest.json,1631193298765,d68cf3228ef6d6553c8298e03cdb9b901d043404b1cc88242a92e0124523e834
```
Comment (dobryanskyy) [inline .firebase/hosting.YnVpbGQ.cache]: this should be removed

File: src/components/header/accountMenu/accountMenu.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import React, {useState} from 'react';
+import './accountMenu.css';
+import AccountMenuButton from './accountMenuButton';
+import AccountMenuWindow from './accountMenuWindow';
+
+function AccountMenu() {
+    const [toggleOpen, setToggleOpen] = useState(false);
```
Comment (dobryanskyy) [inline src/components/header/accountMenu/accountMenu.js]: open, setOpen

File: src/components/header/accountMenu/accountMenu.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+    function onClick () {
```
Comment (dobryanskyy) [inline src/components/header/accountMenu/accountMenu.js:9]: toggleOpen

File: src/components/header/accountMenu/accountMenuWindowUpperMiddle/accountMenuWindowUpperMiddle.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+import config from './../../../../config';
+import addAccountImg from './../../../../img/addAccount.png';
+
+
+function AccountMenuWindowUpperMiddle() {
+
+    return(
+        <div className="div1-amwum">
+            <a href={config.addAccountLink} className="a-amwum">
+                <div className="div-img-amwum">
```
Comment (dobryanskyy) [inline src/components/header/accountMenu/accountMenuWindowUpperMiddle/accountMenuWindowUpperMiddle.js]: what is amwum?

**Review by dobryanskyy — CHANGES_REQUESTED** [review]

**Pull request merged**